### PR TITLE
Fix for `bsp4j/xtend`

### LINKED ISSUE #255: `sbt bsp4j/xtend` doesn't work
It doesn't work both locally and in CI.
It seems like it broke last/this week according to the [actions](https://github.com/build-server-protocol/build-server-protocol/actions). Looking at the error message it is related to the https://github.com/eclipse/xtext/issues/1231

### PR DESCRIPTION
**Pull request by dos65**
The weird solution that was found by an accident.

Fixes #255

### CODE REVIEW

**Comment by kpodsiad** [comment]
According to the https://www.eclipse.org/xtend/releasenotes.html#/releasenotes/2020/09/01/version-2-23-0 2.23.0 is the last version which supports jdk8. This makes me wonder how `2.25.0` which is the currently used version has worked for that long.

Maybe downgrading to the `2.23.0` + your fix is a workaround which is needed

**Review by kpodsiad — COMMENTED** [review]
Shouldn't `bsp4j` have a setting that sets the targeted java version to jdk8? Now it depends on jdk version with which sbt was runned.

File: .github/workflows/verify.yml [hunk 1]
```diff
@@ -6,17 +6,12 @@ on:
...
-    strategy:
-      fail-fast: false
-      matrix:
-        java: [adopt@1.8, adopt@1.11]
     steps:
       - uses: actions/checkout@v2
       - uses: olafurpg/setup-scala@v11
         with:
-          java-version: ${{ matrix.java }}
+          java-version: adopt@1.11
```
[the remark this answers is not in the document]
Comment (dos65) [reply .github/workflows/verify.yml:14]: I think as long as we support jdk8 it's better to use it instead of greater versions

**Comment by dos65** [comment]
@kpodsiad I've tried to downgrade versions but it doesn't help.

@jastice could we merge this PR? I'm out of ideas how to fix that. I hope having this ugly fix should be enough for a while

**Comment by jastice** [comment]
Thanks for this! Hope this doesn't break anything 🙈

**Pull request merged**